### PR TITLE
Command line instructions

### PR DESCRIPTION
**Pull request by shaunluttin**

### CODE REVIEW

File: samples/CodeFlow/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+Start the authorization server from its directory.                 
+```                                                                
+cd AuthorizationServer                                             
+dotnet run -f netcoreapp1.0 --server.urls="http://localhost:54540" 
+```                                                                
+                                                                   
+Then, run the client application from its directory.               
+```                                                                
+cd ClientApp                                                       
+dotnet run -f netcoreapp1.0  --server.urls="http://localhost:53507"
```
Comment (shaunluttin) [inline samples/CodeFlow/README.md]: Delete extra space.

File: samples/ClientCredentialsFlow/ClientApp/Program.cs [hunk 2]
```diff
@@ -15,14 +16,26 @@ public class Program
...
+            catch (System.Net.Http.HttpRequestException ex)
+            {
+                var builder = new StringBuilder();
+                builder.AppendLine("+++++++++++++++++++++");
+                builder.AppendLine(ex.Message);
+                builder.AppendLine(ex.InnerException.Message);
+                builder.AppendLine("Make sure you started the authorization server.");
+                builder.AppendLine("+++++++++++++++++++++");
+                Console.WriteLine(builder.ToString());
+            }
```
Comment (shaunluttin) [inline samples/ClientCredentialsFlow/ClientApp/Program.cs]: If you like this, let me know, and I will add it also to the password flow sample.

Comment (PinpointTownes) [inline samples/ClientCredentialsFlow/ClientApp/Program.cs]: Sounds like a good idea :+1:

**Comment by PinpointTownes** [comment]
Thanks :+1:

**Pull request merged**